sdk/analytics/src/main/java/com/klaviyo/analytics/DeviceProperties.kt

```diff
@@ -48,11 +48,11 @@ internal object DeviceProperties {
     }
 
     val sdkVersion: String by lazy {
-        BuildConfig.VERSION
+        KlaviyoConfig.sdkVersion
```

Comment:
might not want these to be lazy anymore, i think if you were to re-init and change the version number or sdk name for some reason, it would not get reflected here.

Reply:
^ if this is a concern you could do something like this:
```suggestion
val sdkVersion: String
        get() = KlaviyoConfig.sdkVersion
```

Reply:
Thanks makes sense. Thanks @evan-masseau updated it to be non-lazy. Thanks for the code sample @dan-peluso